website/content/en/preview/concepts/disruption.md

```diff
@@ -227,7 +212,7 @@ Voluntary node removal does not include [Interruption]({{<ref "#interruption" >}
 
 ### Node-Level Controls
 
-Nodes can be opted out of consolidation deprovisioning by setting the annotation `karpenter.sh/do-not-consolidate: "true"` on the node.
+Nodes can be opted out of consolidation disruption by setting the annotation `karpenter.sh/do-not-consolidate: "true"` on the node.
```

Comment:
This should be do-not-disrupt, no?
```suggestion
Nodes can be opted out of consolidation disruption by setting the annotation `karpenter.sh/do-not-disrupt: "true"` on the node.
```

Reply:
Good catch. This is what I get for trying to do a find and replace 😝